sonar-ws/src/main/java/org/sonarqube/ws/client/HttpConnector.java

```diff
@@ -192,16 +192,20 @@ private WsResponse post(PostRequest postRequest) {
     return okHttpRequestBuilder;
   }
 
-  private HttpResponse doCall(Request okRequest) {
+  private OkHttpResponse doCall(Request okRequest) {
     Call call = okHttpClient.newCall(okRequest);
     try {
       Response okResponse = call.execute();
-      return new HttpResponse(okResponse);
+      return new OkHttpResponse(okResponse);
     } catch (IOException e) {
       throw new IllegalStateException("Fail to request " + okRequest.urlString(), e);
     }
   }
 
+  public static Builder builder() {
```

Comment:
the constructor was kept public for backward-compatibility, but it's not a good reason.